### DISCUSSION TITLE
What to do when the library depends on `process.stdout`?

### DISCUSSION THREAD

**Discussion by woniesong92** [vercel/next.js#17593] [Help]
I am trying to use [ffmpeg.js](https://github.com/Kagami/ffmpeg.js/) with next.js. But I run into an exception because the library depends on `process.stdout` to exist. I was able to solve a similar problem where the library depended on `fs` simply by setting the config this way [following this suggestion](https://github.com/vercel/next.js/issues/7755#issuecomment-508633125):

```
module.exports = {
  webpack: (config, { isServer }) => {
    // Fixes npm packages that depend on `fs` module
    if (!isServer) {
      config.node = {
        fs: "empty",
      };
    }

    return config;
  },
};
```

Is there a similar step I can follow to work around the dependency on `process.stdout`?

**Comment by jamesmosier** [comment]
If the package relies on `process.stdout` that means it should be running in a Node environment on a server, not your browser. So polyfilling that is not a great option because there are likely other Node specific internals that would be used.
